common/src/main/java/com/wynntils/core/components/Models.java

```diff
@@ -81,6 +82,7 @@ public final class Models {
     public static final TerritoryModel Territory = new TerritoryModel();
     public static final TokenModel Token = new TokenModel();
     public static final TowerAuraTimerModel TowerAuraTimer = new TowerAuraTimerModel();
+    public static final TrackerModel Tracker = new TrackerModel();
```

Comment:
Perhaps rename to `ContentTrackerModel`? It is not clear without context. Even with that added name, it is still so-so..

Reply:
Renamed. I hope I got all places. I have not renamed all internal variables, it got too unwieldy, and in those cases, I think it is clear from the context.